mujoco_warp/_src/types.py

```diff
@@ -62,6 +62,22 @@ class BlockDim:
   mul_m_dense: int = 32
 
 
+class BroadphaseFilter(enum.IntFlag):
+  """Broadphase filter.
```

Comment:
nit, I think we can be a bit more descriptive:

```py
class BroadphaseFilter(enum.IntFlag):
  """Bitmask specifying which collision functions to run during broadphase.
  ...
  """
```

Reply:
done